lua/lint/linters/protolint.lua

```diff
@@ -12,7 +12,7 @@ return {
     end
     local json_output = vim.json.decode(output)
     local diagnostics = {}
-    if json_output.lints == nil then
+    if json_output.lints == nil or json_output.lints == vim.NIL then
```

Comment:
Just fyi: There's also `vim.json.decode(output, { luanil = { object = true }})` - which can be useful if `null` appears often in the json and one doesn't need to distinguish between absent properties/null.

Reply:
Oh, nice - I didn't know. Good to know, thanks!